lodash.js

```diff
@@ -3958,11 +3960,26 @@
      *
      * @private
      * @param {ArrayBuffer} arrayBuffer The array buffer to clone.
+     * @param {Number} [offset] Zero-based index at which to slice.
      * @returns {ArrayBuffer} Returns the cloned array buffer.
      */
-    function cloneArrayBuffer(arrayBuffer) {
-      var result = new arrayBuffer.constructor(arrayBuffer.byteLength);
-      new Uint8Array(result).set(new Uint8Array(arrayBuffer));
+    function cloneArrayBuffer(arrayBuffer, offset) {
+      offset = typeof offset === 'undefined' ? 0 : offset;
+      var result = new arrayBuffer.constructor(arrayBuffer.byteLength, offset);
+      new Uint8Array(result).set(new Uint8Array(arrayBuffer, offset));
+      return result;
+    }
+
+    /**
+     * Creates a clone of `dataView`.
+     *
+     * @private
+     * @param {DataView} dataView The data view to clone.
+     * @returns {DataView} Returns the cloned data view.
+     */
+    function cloneDataView(dataView) {
+      var buf = cloneArrayBuffer(dataView.buffer, dataView.byteOffset),
```

Comment:
Also at first I thought I'd just make `cloneArrayBuffer` use `TypedArray.slice` - it seemed like an efficient way to get the data. 
@jdalton  I am guessing it isn't used because of IE10 support?